web/src/routes/(user)/photos/[[assetId=id]]/+page.svelte

```diff
@@ -91,18 +128,35 @@
   </AssetSelectControlBar>
 {/if}
 
-<UserPageLayout hideNavbar={$isMultiSelectState} showUploadButton scrollbar={false}>
-  <AssetGrid
-    enableRouting={true}
-    {assetStore}
-    {assetInteractionStore}
-    removeAction={AssetAction.ARCHIVE}
-    on:escape={handleEscape}
-    withStacked
-  >
-    {#if $preferences.memories.enabled}
-      <MemoryLane />
+<UserPageLayout
+  showCustomSidebar={options.previousRoute !== undefined || options.coordinates !== undefined}
```

Comment:
Can we extract this to a variable to show the meaning behinds the logic?

Reply:
Done